src/index.html

```diff
@@ -22,6 +22,367 @@
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a
+        class="header__logo-link"
+        href="https://www.google.com"
+      >
+        <img
+          class="header__logo-image"
+          src="images/logo.png"
+          alt="logo"
+        />
+      </a>
+
+      <nav class="header-nav">
+        <ul class="header-nav__list">
+          <li class="header-nav__item">
+            <a
+              class="header-nav__item-link is-active"
+              href="#apple"
+            >
+              Apple
+            </a>
+          </li>
```

Comment:
Fix classes everywhere
```suggestion
      <nav class="nav">
        <ul class="nav__list">
          <li class="nav__item">
            <a
              class="nav__link is-active"
              href="#apple"
            >
              Apple
            </a>
          </li>
```